Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte

```diff
@@ -49,15 +49,58 @@
 
   const docIsDeleted = $keyValueStore["currentDoc"].status === "deleted";
 
+  const createOnWooCommerceAndUpdateInDb = (doc) =>
+    woocommerceClient
+      .createItem(doc)
+      .then((wcDoc) => {
+        doc.wc_url = wcDoc.permalink;
+        doc.wc_id = wcDoc.id;
+        Database.updateDoc(doc);
+        notifier.success("Gegenstand auf der Webseite erstellt!", 3000);
+      })
+      .catch((error) => {
+        notifier.warning("Gegenstand konnte auf der Webseite nicht erstellt werden!", 6000);
+        console.error(error);
+      });
+
   keyValueStore.setValue("mock", {
     status: "gelöscht",
   });
 
   const popupFormularConfiguration = new PopupFormularConfiguration()
     .setTitle(`Gegenstand ${createNew ? "anlegen" : "bearbeiten"}`)
-    .setDisplayDeleteButton(!createNew)
-    .setInputGroups(["Bezeichnung", "Beschreibung", "Eigenschaften", "Bild", "Status"])
+    .setDisplayDeleteButton(!createNew && !docIsDeleted)
+    .setDisplaySaveButton(!docIsDeleted)
+    .setInputGroups([
+      ...(docIsDeleted ? ["Wie­der­her­stel­len"] : []),
+      "Bezeichnung",
+      "Beschreibung",
+      "Eigenschaften",
+      "Bild",
+      "Status",
+    ])
     .setInputs([
+      ...(docIsDeleted
+        ? [
+            {
+              type: InputTypes.BUTTON,
+              group: "Wie­der­her­stel­len",
+              text: "Gelöschten Gegenstand wie­der­her­stel­len",
+              onClick: async () => {
+                if (
+                  confirm(
+                    "Soll dieser Gegenstand in der Datenbank und auf der Webseite wiederhergestellt werden?"
```

Comment:
Why mention the database here?

Reply:
My intention was to make clear that the item will be restored in the database (where it is currently marked as deleted) and in WooCommerce. Would you prefer another message?

Reply:
I think it's good to mention both!

Reply:
I would just mention that it is restored. Mentioning both confuses people that do not have that much insight into the inner workings of the application and none of the other messages seems to mention the database. Besides, "database" might be too generic.